keyboards/helix/pico/rules.mk

```diff
@@ -1,2 +1,19 @@
+KEYBOARD_LOCAL_FEATURES_MK := $(dir $(lastword $(MAKEFILE_LIST)))local_features.mk
+
+SRC += local_drivers/i2c.c
```

Comment:
```suggestion
QUANTUM_LIB_SRC += local_drivers/i2c.c
```

Reply:
Of course, I know about the LIB_SRC and QUANTUM_LIB_SRC macros.

However, with this pull-request we/I want to make sure that the generated binary does not change.

If I use LIB_SRC, the generated binary will change as the link order changes. So I didn't use them.

Reply:
While that may be true, part of the reason for the LIB SRC stuff was to avoid issues.   Like, I had issues with my corne because I have LTO enabled, and using these fixed the issue.   So optimizing these files seem to lead to communication issues.

However, if you're fine with leaving these as just `SRC`, that's fine.

Reply:
I don't know now whether Helix and Helix pico have problems when compiling with LTO. If something goes wrong, someone will need to fix and open a new Pull-Request.

This Pull-Request is not intended to solve such unknown or known problems. I also want to avoid creating new problems with this Pull-Request. Therefore, I hope that the binary generated by this Pull-Request will not be changed.